frontend/src/components/program/header/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+import Nav from '../../header/nav/index';
+
+class Header extends React.Component {
+  render() {
+    return (
+      <React.Fragment>
+        <section className="our-program__header u-content-wrapper">
+          <Nav />
+          <div className="our-program__header-content u-content">
+            <h1 className="our-program__header-content__title">
+              OUR PROGRAM
+            </h1>
+            <p className="our-program__header-content__description">
+              You may be skilled in your field, but when you start your career, <br />
```

Comment:
I think this might look awkward on some screen sizes, so I'd remove the `<br />`. 

General tip I've been learning: when designers give their templates, I don't stick 1:1 to their proposals, because they don't usually consider different screen sizes. It's the intent that's important for me: Their intent probably was: Hey here we have padding /margin and whenever the line breaks, breaks, but me as designer would make it really slick 
break wherever it looks most beautiful. What I mean is: It's ok if we don't break the line on the `,` on each screen size, because on most of them it would look awkward. There would be a particular screen size that the line breaks there just because of spacing and that would be the screen size that the designer picked, because it looked best.